src/Libraries/Microsoft.Extensions.Http.Diagnostics/Microsoft.Extensions.Http.Diagnostics.csproj

```diff
@@ -43,9 +43,7 @@
     <PackageReference Include="Microsoft.IO.RecyclableMemoryStream" />
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" />
     <PackageReference Include="Microsoft.Extensions.Http" />
-    <PackageReference Include="System.Text.Json" />
     <PackageReference Include="System.Collections.Immutable" Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))" />
-    <PackageReference Include="System.Net.Http" Condition="'$(TargetFramework)' == 'net462'" />
```

Comment:
Does the analyzer consider the cases when references are conditioned like this one? I assume it doesn't run against all cross-compiled tfms?

Reply:
I think it does, the analyzer warnings vary based on TFM.